### PR TITLE
Add tabs to options.

### PR DESCRIPTION
**Pull request by reiclone**
From #42.

### CODE REVIEW

**Comment by bakape** [comment]
Hmm, the colour of the selected tab needs to be consistent to the theme. Fixing the padding would probably require playing with `-webkit-padding-*` and `-moz-padding-*`. Also, keyboard shortcuts should probably be made a tab of it's own. That probably means removing it's existing toggleability. Will also prevent bugs like bellow. Otherwise, looks wonderful. If you have a better idea of how to solve any of these do tell. 
google plz1

**Comment by reiclone** [comment]
I'm going to do what you said, just a question.
What do you mean with the wrong padding? do you get different padding in firefox and chrome?

**Comment by bakape** [comment]
The extra padding on the right. Purely aesthetic though.

File: client/options.js [hunk 1]
```diff
@@ -750,7 +760,9 @@ if (!localStorage.getItem('options')){
 		$('#options').removeClass('noOptions');
 	});
 }
-
+function add_shortcuts() {
```
Comment (bakape) [inline client/options.js]: Removed this in the commit.

**Comment by bakape** [comment]
Perfect. Thanks!

**Pull request closed without merging**